symfony/phpunit-bridge/3.3/tests/bootstrap.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+require __DIR__.'/../vendor/autoload.php';
+
+use Symfony\Component\Dotenv\Dotenv;
+
+/*
+ * Environment variables can also be specified in phpunit.xml.dist.
+ * Those variables will override any defined in .env.
+ */
+if (!isset($_SERVER['APP_ENV'])) {
```

Comment:
Shouln't this be the same as for version 4.7?

`if (!isset($_SERVER['APP_ENV']) || 'prod' !== $_SERVER['APP_ENV']) {`